src/core/session.py

```diff
@@ -0,0 +1,32 @@
+# coding: utf8
+
+__contact__ = "info@hytech-imaging.fr"
+__copyright__ = "Copyright (c) 2021 Hytech Imaging"
+
+from qgis._core import QgsProject
+from .database import SammoDataBase
+
+
+class QgsMapLayerRegistry:
+    pass
```

Comment:
Is it useful?

Reply:
?? I think it is an automatic creation from my IDE.
Done 6th commit (f9099ae08ce5c3734cf4631a5e052f67b1a83ea9)